userspace/falco/app/actions/helpers_inspector.cpp

```diff
@@ -54,6 +54,11 @@ falco::app::run_result falco::app::actions::open_live_inspector(
 {
 	try
 	{
+		if((s.config->m_metrics_flags & PPM_SCAP_STATS_STATE_COUNTERS))
```

Comment:
If we enable these stats with a source plugin should we expect all empty values?